packages/mobile/src/app/WebViewScreen.tsx

```diff
@@ -129,13 +126,13 @@ function WebViewScreen({ route, navigation }: Props) {
         ref={webViewRef}
         originWhitelist={['https://*', 'celo://*']}
         onShouldStartLoadWithRequest={handleLoadRequest}
-        setSupportMultipleWindows={false}
         source={{ uri }}
         startInLoadingState={true}
         renderLoading={() => <ActivityIndicator style={styles.loading} size="large" />}
         onNavigationStateChange={(navState) => {
           setCanGoBack(navState.canGoBack)
           setCanGoForward(navState.canGoForward)
+          handleSetNavigationTitle(navState.url, navState.title)
```

Comment:
i left the native browser behaviour here - while the website is loading, we don't know the title so it defaults to the url. this is why in the video we see a flash of the url before the title is displayed. this also happens on browsers like Chrome so i thought it was correct to leave it like this

Reply:
nit: Ok, I wonder if we should remove the title, or display "Loading..." instead of "https://..." 🤔 
I've tried on mobile with: Safari (iOS), and Chrome (iOS and Android).
They don't display the title.

We can check with Nitya when she's available.